api/Document.json

```diff
@@ -8281,10 +8277,12 @@
               ]
             },
             "samsunginternet_android": {
-              "version_added": "2.0"
+              "version_added": "8.0",
+              "notes": "Before Samsung Internet 8.0, this event handler attribute was not supported; however, the event itself was supported since Samsung Internet 2.0. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
```

Comment:
```suggestion
              "notes": "Before Samsung Internet 8.0, this event handler attribute is not supported; however, the event itself is supported since Samsung Internet 2.0. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
```